### ISSUE TITLE
Error building latest vcpkg (Using bootstrap bat)

### ISSUE THREAD

**Issue by akoolenbourke** [Microsoft/vcpkg#1908]
Hi all, currently running 
```
Vcpkg package management program version 0.0.81-b33632943c51a53c61fbbfe38b4f23c1767dc602
```
I ran bootstrap-vcpkg.bat and building resulted in:
```
        commands_search.cpp
     3>..\src\commands_integrate.cpp(120): error C2440: '=': cannot convert from 'const char [6]' to 'LPCWSTR' [D:\src\vcpkg\toolsrc\vcpkglib\vcpkglib.vc
       xproj]
         ..\src\commands_integrate.cpp(120): note: Types pointed to are unrelated; conversion requires reinterpret_cast, C-style cast or function-style c
         ast
     3>..\src\commands_integrate.cpp(121): error C2440: '=': cannot convert from 'const char [4]' to 'LPCWSTR' [D:\src\vcpkg\toolsrc\vcpkglib\vcpkglib.vc
       xproj]
         ..\src\commands_integrate.cpp(121): note: Types pointed to are unrelated; conversion requires reinterpret_cast, C-style cast or function-style c
         ast
     3>..\src\commands_integrate.cpp(123): error C2440: '=': cannot convert from 'const char *' to 'LPCWSTR' [D:\src\vcpkg\toolsrc\vcpkglib\vcpkglib.vcxp
       roj]
         ..\src\commands_integrate.cpp(123): note: Types pointed to are unrelated; conversion requires reinterpret_cast, C-style cast or function-style c
         ast
     3>..\src\commands_integrate.cpp(128): error C2664: 'BOOL ShellExecuteExA(SHELLEXECUTEINFOA *)': cannot convert argument 1 from 'SHELLEXECUTEINFO *'
       to 'SHELLEXECUTEINFOA *' [D:\src\vcpkg\toolsrc\vcpkglib\vcpkglib.vcxproj]
         ..\src\commands_integrate.cpp(128): note: Types pointed to are unrelated; conversion requires reinterpret_cast, C-style cast or function-style c
         ast
```

Any ideas?
Cheers

**Comment by ras0219-msft** [comment]
Fixed in master (20d911a9). Thanks for the PR!

**Comment by akoolenbourke** [comment]
Awesome. Thanks for the fast response and it worked a treat. And thanks for abseil support too.
